final/README.md

```diff
@@ -0,0 +1,447 @@
+    * A foglalásra vonatkozó szabályokról (pl. az ár kiszámítása) bővebben olvashatsz a következő szekciókban.
+    * Például:
+    ```
+    Signed in with account 'sanyi'
+    Your previous bookings are
+    Seats (5,5), (5,6) on Sátántangó in room Pedersoli starting at 2021-03-15 10:45 for 3000 HUF
+    ```
+* A parancs kimenete más esetekben a kettes jegyhez szükséges követelmények között van leírva.
+
+##### Jegyfoglalás
+
+###### Foglalás létrehozása
+* A következő parancs lehetőséget biztosít egy már meglévő vetítésre, egy vagy több ülőhelyre
+jegyet foglalni.
+```
+book <film címe> <terem neve> <vetítés kezdetének ideje YYYY-MM-DD hh:mm formátumban> <a lefoglalandó ülőhelyek listája, szóközzel elválasztva, egy-egy ülőhely "(<sor>,<oszlop>)" formátumú> 
+```
+Például:
+```
+book Sátántangó Pedersoli "2021-03-15 10:45" (5,5) (5,6)
+```
+* A foglalás sikertelen akkor, ha 
+    * A megjelölt ülőhelyek közül legalább egy már foglalt. Ekkor a parancs kimenete
+    ```
+    Seat <ülőhely> is already taken
+    ```
+    ahol az ülőhely a listában megadottak közül az első olyan, amely nem foglalható
+    * A megjelölt ülőhelyek valamelyike nem létezik az adott teremben. Ekkor a parancs kimenete
+    ```
+    Seat <ülőhely> does not exist in this room
+    ```
+    ahol az ülőhely a listában megadottak közül az első olyan, amely nem foglalható a hiba miatt.
+* Amennyiben a foglalás sikeres, a parancs kimenete a következő
+```
+Seats booked: <a lefoglalt ülőhelyek listája, ", "-el elválasztva, egy-egy ülőhely "(<sor>,<oszlop>)" formátumú>; the price for this booking is <a jegy ára> HUF 
+```
+    * A jegy ára 1500 HUF / ülőhely / vetítés. Az ötös érdemjegyhez tartozó követelményekben 
+    további szabályok kerülnek bevezetésre.
+* Ez a parancs csak bejelentkezett, nem adminisztrátor felhasználóknak elérhető
+
+### For grade 5:
+#### Általános követelmények a forráskóddal és az alkalmazással kapcsolatban
+
+* A négyes érdemjegy eléréséhez tartozó összes követelmény teljesül
+* Az `mvn clean verify -p requirements-grade3` sikeresen lefut 
+a `ticket-service-parent` projekten. Ehhez a négyes érdemjegy eléréséhez szükséges 
+feltételeken túl a következők szükségesek:
+    * Legalább 70%-os unit teszt branch és line coverage az egész `ticket-service` modulra vonatkoztatva.
+    * A `@grade5-requirements` tag-gel megjelölt acceptance tesztekben leírt követelményeknek megfelelő alkalmazás.
+
+#### Az alkalmazás által támogatott parancsok
+
+##### Árazással kapcsolatos funkcionalitás
+
+###### Az alapértelmezett ár megváltoztatása
+* Egy foglalás alapára 1500 HUF / ülőhely / vetítés. 
+A foglalás ára (/ülőhely/vetítés) kiszámítható a foglalás alapárának 
+és a foglalásra vonatkozó árkomponensek összegzésével. Az árkomponensek
+leírását lásd később. 
+* A következő parancs a foglalás alapárának megváltoztatását teszi lehetővé
+```
+update default price <új alapértelmezett ár> HUF
```

Comment:
I guess we can remove the currency; the requirement does not mention handling foreign currency